cmd/eksctl-anywhere/cmd/cleanup.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/aws/eks-anywhere/pkg/types"
 )
 
-func cleanup(ctx context.Context, deps *dependencies.Dependencies, commandErr error) {
+func cleanup(ctx context.Context, deps *dependencies.Dependencies, commandErr *error) {
 	close(ctx, deps)
 
 	if commandErr == nil {
```

Comment:
I could be mistaken but I think we might need `*commandErr` here
I believe the pointer is never going to be nil

or to be safe maybe just `commandErr == nil || **commandErr == nil`

Reply:
I dunno I think this works -- https://go.dev/play/p/kDa2v0oDGMf